.github/actions/upload-sentry-sourcemaps/action.yml

```
@@ -0,0 +1,107 @@
+  steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Validate Sentry auth token
+      shell: bash
+      run: |
+        set -euo pipefail
+        echo "🔐 Validating Sentry authentication token..."
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)